rfcs/PROCESS.md

```diff
@@ -4,6 +4,8 @@ Changes to ECS are proposed as Requests for Comments (RFC) in [rfcs/](./rfcs/) a
 
 Each RFC is represented as a markdown document following a prescribed template that gets committed to the repo. Each stage of the RFC is represented as a pull request against that document.
 
+Every RFC should also have a corresponding folder (named after the RFC number) in [rfcs/text/](./text/). The folder should contain the proposed schema changes as standalone YAML files, or extended example mappings and larger source documents.
```

Comment:
```suggestion
If proposing new fields or changing existing fields, the RFC should also have a corresponding folder (named after the RFC number) in [rfcs/text/](./text/). The folder should contain the proposed schema changes as standalone YAML files or extended example mappings and larger source documents.
```

I think most RFCs will be adds/changes, and having the directory will be necessary. There may also be cases, like removing fields, where it's probably not needed, though. 

Also, a minor grammar nitpick.